packages/core/config/ember-try.js

```diff
@@ -1,83 +1,81 @@
-/* eslint-env node */
-module.exports = {
-  scenarios: [
-    {
-      name: 'ember-lts-2.8',
-      bower: {
-        dependencies: {
-          ember: 'components/ember#lts-2-8'
+'use strict';
```

Comment:
We can probably get rid of this file as well.

Reply:
So we aren't using this file then?  If so I'll scrap it.

Reply:
So we aren't using this file? If so I'll get rid of it.

Reply:
No, even if we do, i think we might end up having a global ember try.